# Cancer Omics Analysis

- [Cancer Omics Analysis](#cancer-omics-analysis)
  - [Introduction](#introduction)
  - [Objective](#objective)
  - [Dataset](#dataset)
  - [Literature](#literature)
    - [Reviews on DNA Methylation data](#reviews-on-dna-methylation-data)
    - [Review on Hematopoiesis](#review-on-hematopoiesis)
  - [How to structure your project](#how-to-structure-your-project)
    - [Project proposal](#project-proposal)
    - [Project](#project)

## Introduction

While DNA Methylation is a key mechanism influencing transcriptional processes, it be used to differentiate and classify cells. Especially in developmental processes such as haematopoesis, DNA methylation is a vital process influencing cell fate. Both array based and sequencing based methods can provide a map of DNA methylation, with a varying degree of coverage.

Here, you are tasked with identifying differentially methylation regions between two groups of samples and interpret the sequence context of these regions. While the general state of methylation in these regions can be of interest, for comparison of cell populations finding and reducing the regions of interest is vital. Further, comparative analysis between malignant and healthy cells provides the basis for identifying pathways ang genes affected.

## Objective

A Variety of tools exist 

- Choose 

## Dataset

## Literature

### Reviews on DNA Methylation data

- [Analysing and interpreting DNA methylation data](https://www.nature.com/articles/nrg3273)
- [Statistical and integrative system-level analysis of DNA methylation data](https://www.nature.com/articles/nrg.2017.86)

### Review on Hematopoiesis
- 

## How to structure your project

### Project proposal

### Project